src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -38,11 +38,24 @@ public function prepend(ContainerBuilder $container)
             return;
         }
 
-        if (!isset($frameworkConfiguration['serializer'], $frameworkConfiguration['serializer']['enabled'])) {
+        $serializerConfig = null;
+        $propertyInfoConfig = null;
+
+        foreach ($frameworkConfiguration as $frameworkParameters) {
+            if (isset($frameworkParameters['serializer'])) {
+                $serializerConfig = $serializerConfig ?: $frameworkParameters['serializer'];
```

Comment:
You can do `$serializerConfig = $serializerConfig ?? $frameworkParameters['serializer']`. And the 2 vars can be deleted.